docs/contribute-module.md

```diff
@@ -1,81 +1,73 @@
-# Contributing to IBM Cloud Terraform Modules
+# Contributing to the IBM Cloud Terraform modules project
 
-Create a Terraform module. You start from the template module, pull the dependencies, and modify the template Terraform logic.
+Create or update a Terraform module. When you create a module, you start from the template module. When you update a module, you fork the module. Then, modify the Terraform logic.
 
-Following these contribution guidelines helps to communicate that you respect the time of the developers managing. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+The process to create a module has several goals.
 
-The process described here has several goals:
+- To maintain IBM Cloud Terraform modules quality.
+- To fix problems that are important to users.
+- To engage the community in working toward the best possible Terraform modules for IBM Cloud.
 
-- Maintain IBM Cloud Terraform Modules quality
-- Fix problems that are important to users
-- Engage the community in working toward the best possible IBM Cloud Terraform modules
+By following these contribution guidelines, you help to communicate that you respect the time of the maintainers and you reduce the chances that your PR is rejected. In return, the maintainers agree to reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
```

Comment:
I would remove this part - I believe it comes from the original documentation for this org, but this sounds a bit 'passive aggressive' and not really welcoming